### DISCUSSION TITLE
Welcome to SAP BTP Solution Diagrams Discussions!

### DISCUSSION THREAD

**Discussion by fabianleh** [SAP/btp-solution-diagrams#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our [community](https://community.sap.com/). We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members also make use of the [SAP Community](https://community.sap.com/)
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by kurumesa** [comment]
Although I find an icon for Integration Suite, I do not find individual icons for API Management, Cloud Integration, Open Connectors. Similarily, I do not find anything for Cloud Connector. Any suggestions?

**Reply by fabianleh** [reply]
Hi @kurumesa , the Integration Suite related / capabilities will be provided soon. For Cloud Connector we do not have an Icon, we're using this one: 
<img width="450" alt="image" src="https://github.com/SAP/btp-solution-diagrams/assets/91141653/b549e124-4496-474c-af0e-60acb7da25b7">

Hope this helps...
For the Integration Suite capabilities pls. check this blog time by time I will inform there when we have the final SD Icons availble.

Best,
Fabian

**Comment by gabrielagahse** [comment]
The default generic icons library doesn't provide all icons in grey. I am missing the following icons in grey
![Missing](https://github.com/SAP/btp-solution-diagrams/assets/45011462/e6e5c323-ad84-40b9-add3-0a093776fdff)

And is there any chance to add more generic icons? I saw more icons in the Workflow example and wondering how they have used them.

Thanks and best regards, 
Gabriela

**Comment by iamashokkumar** [comment]
Team, forgive me if this isn't the right place for this question/thought.

I’ve often wondered why SAP BTP doesn’t have an official logo or icon like our hyperscaler partners do.
While our ecosystem knows what BTP is, I believe having a distinct icon/logo could further enhance its visibility and impact. What do you think?
I wanted to share this on the internal BTP SharePoint, but since there’s no comment feature, I’m posting here instead!

**Comment by jainatul83** [comment]
@fabianleh fabianleh I am looking for reusable diagram but could not locate it,can you please let me know where I can find them.

**Reply by fabianleh** [reply]
@jainatul83 You can find reusable examples here: https://github.com/SAP/btp-solution-diagrams/tree/main/assets/editable-diagram-examples

**Comment by jainatul83** [comment]
But I am unable to download it ,how to access these files to modified
according to our requirements
Atul Kumar Jain

**Reply by fabianleh** [reply]
@jainatul83 , In the linked md file (https://github.com/SAP/btp-solution-diagrams/blob/main/assets/editable-diagram-examples/README.md), you should be able to click on of the links. This will open the drawio viewer mode, by clicking on the pencil you are able to adopt an architecture to your needs: 
<img width="614" alt="image" src="https://github.com/user-attachments/assets/b1eb9250-fb77-4664-8d62-0c455b78b991">

You can add the "SAP" stencils via the "More Shapes" option:
<img width="616" alt="image" src="https://github.com/user-attachments/assets/ece9895f-87f7-45a5-9b12-05218a87e716">


Hope this helps.

**Comment by kumaresh27** [comment]
Hello,

Wanted to check if we can already start using LucidChart for the diagrams? I do not see any instructions yet on how to get started with the same. The information in https://lucid.app/marketplace#/newlisting/8e327624-ad9a-4ccf-b74c-325bb907a0ef is not very helpful in this regard too. Can you share the latest on this topic?

Br,
Kumaresh

**Comment by saintveni** [comment]
Hi all,

I am not sure, if someone can answer this question and I apologize in advance if this discussion is not the right place for it! 
I am currently working on my PhD thesis and I would like to create the diagrams for it, using the SAP BTP Solution Diagrams.

Does someone know, if I am allowed to?

Thank you in advance!
Best Regards
Nev

**Reply by fabianleh** [reply]
Hi @saintveni ,

no problem ! You can use the BTP SD's for sure, that's the reason why we bring this out as part of SAP's Open Source strategy:
https://github.com/SAP/btp-solution-diagrams?tab=coc-ov-file#readme
You can also use already the direct integration of the SAP shapes in draw.io if not already known: https://sap.github.io/btp-solution-diagrams/docs/solution_diagr_intro/intro/#native-integration-for-the-sap-btp-service-icons--shapes

Hope this helps!?

Best,
Fabian

**Reply by saintveni** [reply]
Hi @fabianleh,

Thank you so much for your prompt feedback! I really appreciate it! 

Best Regards,
Nev

**Comment by MattMadden** [comment]
Hi @fabianleh. I found that two of the icon SVGs, [20076-sap-cloud-transport-management_sd.svg](https://github.com/SAP/btp-solution-diagrams/blob/main/assets/shape-libraries-and-editable-presets/svg/20076-sap-cloud-transport-management_sd.svg) and [31027-sap-document-management-service_sd.svg](https://github.com/SAP/btp-solution-diagrams/blob/main/assets/shape-libraries-and-editable-presets/svg/31027-sap-document-management-service_sd.svg) have been corrupted with instances of "NaN" inside the path data, and I believe this is causing the icons to render incorrectly.

I am attaching an illustration of the issue. On the left side are the icons as provided in the repo, with NaN values in two paths each. On the right are what they look like if I delete the NaN paths altogether. This is obviously an improvement, and probably closer to the intended image. But I'm uncertain if that's exactly how they're supposed to look. Can you help clarify the intended appearance, and perhaps upload corrected versions? Thank yoU!

![image](https://github.com/user-attachments/assets/67eadd94-2c6d-42c2-aed9-46922b5d1a4a)

**Reply by MattMadden** [reply]
Follow up: I just found these [two](https://discovery-center.cloud.sap/serviceCatalog/cloud-transport-management?region=all&tab=service_plan) [pages](https://discovery-center.cloud.sap/serviceCatalog/document-management-service-application-option?service_plan=standard&region=all&commercialModel=btpea) which seem to verify that the corrected images above are, in fact, the correct appearance. I am attaching here the corrected SVG's for your review.


![20076-sap-cloud-transport-management_sd_fixed](https://github.com/user-attachments/assets/b32423c7-9bcb-409c-9d89-72b6d765a502)
![31027-sap-document-management-service_sd_fixed](https://github.com/user-attachments/assets/6a84c5b4-d36b-4f6d-894c-0354a3daf2fb)

**Reply by fabianleh** [reply]
Hi @MattMadden , ah yes identified the same as I working also on updates for Lucid. Will check next week and will update the images/SVG's thx for the hint. 
Best, 
Fabian

**Reply by fabianleh** [reply]
Issue created: https://github.com/SAP/btp-solution-diagrams/issues/183
